docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -419,7 +419,7 @@ This feature is available in Enterprise Edition only.
 1. Run the Deckhouse installer version 1.56.3 or higher.
 
    ```shell
-   docker run -ti --pull=always -v $(pwd)/d8-images:/tmp/d8-images registry.deckhouse.io/deckhouse/ee/install:v1.56.3 bash
+   docker run -ti --pull=always -v $(pwd)/d8-images:/tmp/d8-images registry.deckhouse.io/deckhouse/ee/install:stable bash
```

Comment:
We should not use release channels here in general. Mirror is developed in a faster pace than the deckhouse platform, so we should point users to the most recent released versions of dhctl.

We have plans to move it out from dhctl and into the d8 cli tool in the near future to mitigate the issues with doc pointing to obsolete release number.

Reply:
set alpha